rotkehlchen/exchanges/data_structures.py

```diff
@@ -388,3 +389,33 @@ def trades_from_dictlist(
             continue
 
     return returned_trades
+
+
+binance_pair_db_tuple = Tuple[str, str, str, str]
+
+
+class BinancePair(NamedTuple):
+    """A binance pair. Contains the symbol in the Binance mode e.g. "ETHBTC" and
+    the base and quote assets of that symbol as parsed from exchangeinfo endpoint
+    result"""
+    symbol: str
+    base_asset: Asset
+    quote_asset: Asset
+    location: Location
+
+    def serialize_for_db(self) -> binance_pair_db_tuple:
```

Comment:
Write docstring and include possible exceptions that can be raised

Reply:
I believe no exception can be raised here. Anyway added a note on the fields and why the location field is needed